### PR TITLE
Update stats.py script to use asyncio

### PR DESCRIPTION
**Pull request by elyezer**
The script now runs in about 1/4 of the time it used to run without the
asyncio.

### CODE REVIEW

**Comment by omaciel** [comment]
@rochacbruno wanna chime in?

File: .scripts/stats.py [hunk 1]
```diff
@@ -1,82 +1,98 @@
...
 
     archive_episode_name = match.groups()[0]
     url = "https://archive.org/details/{}&output=json".format(archive_episode_name)
     return (archive_episode_name, url)
 
 
+def print_summary(stats):
+
+    total = sum([v['downloads'] for v in stats.values()])
+    average = float(total)/len(stats)
```
Comment (rochacbruno) [inline .scripts/stats.py]: space around?
`float(total) / len(stats)`

**Comment by rochacbruno** [comment]
@elyezer that script runs on travis?

**Comment by elyezer** [comment]
@rochacbruno not yet, but eventually will

**Comment by omaciel** [comment]
@rochacbruno you got last ACK :)

**Pull request merged**